Examples/Physics_applications/capacitive_discharge/README.md

```diff
@@ -0,0 +1,8 @@
+The examples in this directory are based on the benchmark cases from Turner et
+al. (Phys. Plasmas 20, 013507, 2013). Input files for all four cases are given
+as well as an input file for a shortened simulation used in testing the MCC
+functionality.
+The published results from Turner et al. are provided in the folder
+reference_results/Turner_results/
+Results from running the 4 input cases in WarpX on (insert date) is also
```

Comment:
Just leaving a comment here to make sure the reference results get dated. It would be ideal if they get added/updated in the same commit that fixes this, with the code having been run on the preceding commit, so that there's a very definite reference point.